presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -50,6 +52,7 @@
     private boolean closed;
     private boolean finishing;
     private long joinPosition = -1;
+    private int joinSourcePositions = 0;
```

Comment:
No, actually it is build side positions. I'm interested in counting for a given probe row how many matched build rows we have? It is called LookupSource in this operator, this is why the names.